scripts/jenkins-support.sh

```diff
@@ -36,10 +36,13 @@ function connect_rally_server_run_test
 EOF
 }
 
-function onadmin_install_ca_certificates
+function install_ca_certificates_on_cloud
 {
```

Comment:
Onadmin means it will be run on the crowbar node. This also controls how this function is called. Please leave onadmin.